File: matrixElementsSum.js
function matrixElementsSum(matrix) {
    roomSum = 0;
    for (i = 0; i < matrix[0].length; i++) {
        columnIterator = 0;
        while (columnIterator < matrix.length && matrix[columnIterator][i] != 0){
            roomSum += matrix[columnIterator][i];
            columnIterator++;
        }
    }
    return roomSum;
}

File: intro_algorithms/matrixElementsSum/matrixElementsSum.py
def matrixElementsSum(matrix):
    roomSum = 0
    for i in range(len(matrix[0])):
        col = 0
        while col < len(matrix) and matrix[col][i] != 0:
            print(matrix[col][i], " ")
            roomSum += matrix[col][i]
            col += 1
    return roomSum;